regexes/device/mobiles.yml

```diff
@@ -1910,6 +1923,22 @@ CUBOT:
     - regex: '([^;/]+) Build/CUBOT'
       model: '$1'
 
+# Concord
+Concord:
+  regex: 'Concord'
```

Comment:
We should use a more specific regex here, as otherwise it might match some devices of `ConCorde`. Guess a `Concord[ _-]` would be enough